src/entity/user/UserStreakAction.ts

```diff
@@ -0,0 +1,25 @@
+import { Entity, JoinColumn, OneToOne, PrimaryColumn } from 'typeorm';
+import { User } from './User';
+
+export enum UserStreakActionType {
+  Recover = 'recover',
+}
```

Comment:
Let's move this to common/streak or something, it will help with circular dependencies because entities are imported quite a lot all over the palce.

Reply:
Agreed. Let's move it.

Reply:
Just realized, it might actually produce circular dependency if we move it. Since this entity itself needs the enum, then on the same common/users file, we might call the actual entity for fetching.

Reply:
hmm yeah, let's leave it as is then, you are right